autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -465,12 +465,12 @@ def _get_base_search_space(
     def _check_search_space_updates(self, include: Optional[Dict[str, Any]],
                                     exclude: Optional[Dict[str, Any]]) -> None:
         assert self.search_space_updates is not None
+        ignore_updates = []
         for update in self.search_space_updates.updates:
             if update.node_name not in self.named_steps.keys():
                 raise ValueError("Unknown node name. Expected update node name to be in {} "
                                  "got {}".format(self.named_steps.keys(), update.node_name))
             node = self.named_steps[update.node_name]
-            node_name = node.__class__.__name__
```

Comment:
It was kinda better to have it as `node_name`, it is shorter, instead of putting the whole long version 4 times.

Reply:
its an unintended code change